assets/airports/kbos.json

```diff
@@ -398,316 +528,8175 @@
-                "BURDY": ["BURDY"],
-                "BOSOX": ["BOSOX"],
-                "NELIE": ["NELIE"],
                 "GLYDE": ["GLYDE"],
-                "HANAA": ["HANAA"]
+                "HYLND": ["HYLND"],
+                "LBSTA": ["LBSTA"],
+                "LFV": ["LFV"],
+                "LUCOS": ["LUCOS"],
+                "MHT": ["MHT"],
+                "NELIE": ["NELIE"],
+                "ONEPS": ["ONEPS"],
+                "PATSS": ["PATSS"],
+                "PSM": ["PSM"],
+                "REVSS": ["REVSS"],
+                "SEY": ["SEY"],
+                "SPENO": ["SPENO"],
+                "SSOXS": ["SSOXS"]
 
             },
             "draw": [
-                ["BOS", "MHT*"],
-                ["BOS", "ENE*"],
-                ["BOS", "PSM*"],
-                ["BOS", "LFV*"],
-                ["BOS", "ACK*"],
-                ["BOS", "MVY*"],
-                ["BOS", "PVD*"],
-                ["BOS", "SEY*"],
-                ["BOS", "CMK*"],
-                ["BOS", "BAF*"],
-                ["BOS", "CTR*"],
-                ["BOS", "CAM*"],
-                ["BOS", "FRILL*"],
-                ["BOS", "FREDO*"],
-                ["BOS", "DUNKK*"],
-                ["BOS", "BURDY*"],
-                ["BOS", "BOSOX*"],
-                ["BOS", "NELIE*"],
-                ["BOS", "GLYDE*"],
-                ["BOS", "HANAA*"]
+                ["_BOS261002", "WYLYY", "CYOTI"],
+                ["ACK*"], ["BAF*"], ["BLZZR*"],
+                ["BOSOX*"], ["BRUWN*"], ["BURDY*"],
+                ["CELTK*"], ["CTR*"], ["FRILL*"],
+                ["GLYDE*"], ["HYLND*"], ["LBSTA*"],
+                ["LFV*"], ["LUCOS*"], ["MHT*"],
+                ["NELIE*"], ["ONEPS*"], ["PATSS*"],
+                ["PSM*"], ["REVSS*"], ["SEY*"],
+                ["SPENO*"], ["SSOXS*"]
             ]
         }
     },
     "stars": {
+        "GDM4": {
+            "icao": "GDM4",
+            "name": "Gardner Four",
+            "entryPoints": {
+                "ALB": ["ALB", "WIKID"]
+            },
+            "body": ["GDM", "BRONC", "LOBBY", "REVER", "BOS"],
+            "rwy": {
+                "4L": [],
+                "4R": [],
+                "15R": [],
+                "22L": [],
+                "22R": [],
+                "27": [],
+                "32": [],
+                "33L": []
+            },
+            "draw": [
+                ["ALB", "WIKID", "GDM", "BRONC", "LOBBY", "REVER", "BOS"]
+            ]
+        },
         "JFUND1": {
             "icao": "JFUND1",
             "name": "Jayfund One",
             "entryPoints": {
-                "PONCT": ["PONCT", "STOLI", ["MNSTA", "A250"], ["SEETS", "A210|S290"], ["DUBLE", "A180"], ["BASEZ", "A150"]],
-                "WHATE": ["WHATE", ["ONDEC", "A110+"]],
-                "ONDEC": [["ONDEC", "A110+"]]
+                "PONCT": ["PONCT", ["STOLI", "A320-|S350"], ["MNSTA", "A250|S300"], ["SEETS", "A210|S280"], ["DUBLE", "A170"], ["BASEZ", "A140"]]
```

Comment:
We would include them regardless of whether they're assignable only by ATC, that way if we want to assign those entry points, we are able to. However, for example, we could choose to not include those entry points in the spawn patterns for that sort of reason. Nonetheless, I would not necessarily be inclined to do so, because who's to say whether or not the center was or was not assigning them, at the time we're simulating. But we definitely want to include all portions of all procedures, and whether or not we _use_ them for anything is up to the airport designer.